examples/molecular_dynamics/md_nnps.py

```diff
@@ -133,6 +112,7 @@ def solve(self, t, dt):
     o = p.parse_args()
     get_config().use_openmp = o.openmp
     get_config().use_double = o.use_double
+    get_config().suppress_warnings = o.suppress_warnings
```

Comment:
This repetitive block will also be fixed, maybe overload `ArgumentParser.parse_args` to automatically handle these default options.